docs/conf.py

```diff
@@ -62,7 +69,6 @@
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = ".rst"
```

Comment:
Why is this no longer needed?

Reply:
`.rst` is the [default for vanilla `sphinx`](http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-source_suffix) and [`recommonmark` says you only need to alter `source_suffix` for `sphinx` versions <= 1.3](https://recommonmark.readthedocs.io/en/latest/#getting-started)